src/sentry/integrations/msteams/client.py

```diff
@@ -93,14 +89,52 @@ def send_welcome_message(self, conversation_id):
         self.send_message(conversation_id, payload)
 
 
-class OauthMsTeamsClient(ApiClient):
+# MsTeamsClient is used with the access token and service url as arguments to the constructor
+# It will not handle token refreshing
+class MsTeamsClient(MsTeamsAbstractClient):
```

Comment:
I found all the clients a bit confusing, since there are 4 of them (including the base one) but to me it feels like the most 'externally used' (like outside of the install flow) should be named `MsTeamsClient` since it's inline with the other integrations so I think that would be the currently-named `MsTeamsIntegrationClient`. 

Maybe you can rename this to like `MsTeamsInstallClient` idk

Reply:
@MeredithAnya Renamed to `MsTeamsPreInstallClient`